codegen/testdata/simplegrpc/simplegrpc.proto

```diff
@@ -1,6 +1,6 @@
 syntax  = "proto3";
 
-package simplegrpc;
+package simplepb;
```

Comment:
Why is this changing? Does not appear to be part of sysl-go-comms -> sysl-go rename.

The commit title says it is renaming to simpleproto, but you did simplepb. Hard to review without any details, but this does not seem right.

Reply:
I think the repo has lost a few changes from github.service.anz.

Reply:
Good point, the PR seems to have evolved into renaming the sysl-go-comms as well as making sure that `make test` and `make gen` work correctly. 

I've updated the PR description to better reflect what it contains. 

The rename was required to get the module to import correctly for me, but it could be because I've got a differnt version of the *protoc* tool installed. I just tried reverting that commit and checking if the tests pass, and got an import error
```
go test -cover -tags codeanalysis ./...
go: finding github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/simplegrpc latest
go: finding github.com/anz-bank/sysl-go/codegen/tests latest
go: finding github.com/anz-bank/sysl-go/codegen latest
go: finding github.com/anz-bank/sysl-go/codegen/tests/simplegrpc latest
# github.com/anz-bank/sysl-go/codegen/tests/simplegrpc
package github.com/anz-bank/sysl-go/codegen/tests/simplegrpc (test)
	imports github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/simplegrpc: no matching versions for query "latest"
FAIL	github.com/anz-bank/sysl-go/codegen/tests/simplegrpc [setup failed]
?   	github.com/anz-bank/sysl-go/codegen/grammars	[no test files]
ok  	github.com/anz-bank/sysl-go/codegen/tests/dbendpoints	(cached)	coverage: 52.0% of statements
ok  	github.com/anz-bank/sysl-go/codegen/tests/multigrpc	0.024s	coverage: [no statements]
?   	github.com/anz-bank/sysl-go/codegen/tests/multigrpc/callback	[no test files]
?   	github.com/anz-bank/sysl-go/codegen/tests/multigrpc/cards	[no test files]
?   	github.com/anz-bank/sysl-go/codegen/tests/multigrpc/cardspb	[no test files]
?   	github.com/anz-bank/sysl-go/codegen/tests/multigrpc/wallet	[no test files]
ok  	github.com/anz-bank/sysl-go/codegen/tests/simple	0.026s	coverage: 37.9% of statements
?   	github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/callback	[no test files]
?   	github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/simple	[no test files]
?   	github.com/anz-bank/sysl-go/codegen/tests/simplegrpc/simplepb	[no test files]
?   	github.com/anz-bank/sysl-go/codegen/transforms	[no test files]
ok  	github.com/anz-bank/sysl-go/common	(cached)	coverage: 70.1% of statements
ok  	github.com/anz-bank/sysl-go/common/internal	(cached)	coverage: 87.2% of statements
ok  	github.com/anz-bank/sysl-go/config	(cached)	coverage: 70.8% of statements
ok  	github.com/anz-bank/sysl-go/convert	(cached)	coverage: 78.4% of statements
ok  	github.com/anz-bank/sysl-go/core	(cached)	coverage: 38.5% of statements
?   	github.com/anz-bank/sysl-go/core/build	[no test files]
?   	github.com/anz-bank/sysl-go/database	[no test files]
?   	github.com/anz-bank/sysl-go/handlerinitialiser	[no test files]
ok  	github.com/anz-bank/sysl-go/logging	(cached)	coverage: 58.3% of statements
?   	github.com/anz-bank/sysl-go/metrics	[no test files]
ok  	github.com/anz-bank/sysl-go/restlib	(cached)	coverage: 60.9% of statements
ok  	github.com/anz-bank/sysl-go/splunk	(cached)	coverage: 73.3% of statements
?   	github.com/anz-bank/sysl-go/status	[no test files]
ok  	github.com/anz-bank/sysl-go/validator	(cached)	coverage: 85.7% of statements
FAIL
make: *** [test] Error 1
```